src/workerd/io/actor-cache.h

```diff
@@ -702,7 +702,7 @@ class ActorCache final: public ActorCacheInterface {
   kj::Promise<kj::Maybe<Value>> getImpl(kj::Own<Entry> entry, ReadOptions options);
 
   // Ensure that we will flush dirty entries soon.
-  void ensureFlushScheduled(const WriteOptions& options);
+  void ensureFlushScheduled(const WriteOptions& options, kj::LiteralStringConst operationInfo);
```

Comment:
Huh... `LiteralStringConst` wasn't actually meant to be used this way (it's meant purely as a helper for constructing `ConstString`). But I guess it ends up being a convenient way to force people to pass a string literal, whereas accepting `StringPtr` could allow them to pass a string whose lifetime isn't sufficiently long.

I guess that seems fine then. Just hadn't thought of it that way previously.

Now I kind of wish it was just called `LiteralString` instead of `LiteralStringConst` though.